docs/pages/docs/apis/fields.mdx

```diff
@@ -517,6 +517,11 @@ Options:
   This is only allowed when you have no create access control because otherwise, the item will always fail access control
   if a user doesn't have access to create the particular field regardless of whether or not they specify the field in the create.
 
+!> **\*Warning** Unlike with other `keystone` fields, `db.isNullable` is defaulted to `false` for the text field.
+This is primarily in the interest of not having to make assumptions about how a `null` value can be represented in a text field in the Admin-UI in the default case.
+Please our [technical discussion](https://github.com/keystonejs/keystone/discussions/7158) on the topic for an in-depth explanation.
```

Comment:
```suggestion
These differences, and the rational for them, are examined in detail [on GitHub](https://github.com/keystonejs/keystone/discussions/7158).
```

Copy taken from @molomby's change in https://github.com/keystonejs/keystone/pull/7160